steps/pcloudy-app-upload/0.1.2/step.yml

```diff
@@ -13,39 +13,23 @@ source:
 host_os_tags:
 - osx-10.10
 - ubuntu-16.04
-project_type_tags:
-- ios
-- macos
-- android
-- xamarin
-- react-native
-- cordova
-- ionic
 type_tags:
 - test
 toolkit:
   bash:
     entry_file: step.sh
-deps:
-  brew:
-  - name: git
-  - name: wget
-  apt_get:
-  - name: git
-  - name: wget
 is_requires_admin_user: true
 is_always_run: false
 is_skippable: false
 run_if: ""
 inputs:
 - opts:
     description: |
-      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+      The app file you want to upload to Pcloudy, usually $BITRISE\_APK\_PATH or $BITRISE\_IPA\_PATH.
+    is_expand: true
     is_required: true
-    summary: The app file you want to upload to Pcloudy, usually $BITRISE\_APK\_PATH
-      or $BITRISE\_IPA\_PATH
-    title: A Bitrise generated APK or IPA path
-  upload_path: $BITRISE_APK_PATH
```

Comment:
it seems the Step Input key was removed here, without the default value it should look like:

```
- upload_path:
  opts:
     description: |
       The app file you want to upload to Pcloudy, usually $BITRISE\_APK\_PATH or $BITRISE\_IPA\_PATH.
     is_expand: true
     is_required: true
     summary: Location of the IPA or APK that you want to upload.
     title: Location of APK or IPA
```